File: division using function.c
#include <stdio.h>

// Function to divide two numbers
float divide(float a, float b) {
    if(b != 0)
        return a / b;  // Returns quotient
    else {
        printf("Error: Division by zero!\n");
        return 0;
    }
}

int main() {
    float num1, num2, result;

    printf("Enter two numbers: ");
    scanf("%f %f", &num1, &num2);

    // Function call
    result = divide(num1, num2);

    if(num2 != 0)
        printf("Quotient = %.2f", result);

    return 0;
}

File: division using function.py
# Function to divide two numbers
def divide(a, b):
    if b != 0:
        return a / b
    else:
        print("Error: Division by zero!")
        return None

# Input from user
num1 = float(input("Enter first number: "))
num2 = float(input("Enter second number: "))

# Function call
result = divide(num1, num2)

if result is not None:
    print("Quotient =", result)
